fairytale.md

```diff
@@ -26,7 +26,7 @@ and a lot of weights in a series of models.
 
 On this account, I could linger and describe some of the 
 interesting existing models that drive us...
-...would that help understanding more?
+I believe AI will soon surpass human intelligence in many fields, and AI will eventually dominate many industries.
```

Comment:
Hi @stacyyamg ! I will approve this merge as it seems you did right and there's still a lot of students to process, but I wanted to warn you that this change deleted line 29 that was in my main which I usually would not accept. (see https://github.com/floridop/MNXB11-fairytale/pull/109/files#r1796597764)
Merging is not just about choosing one side or the other, sometimes is also about including parts of what others write in the merge you do. 
Good job anyway!